### PR TITLE
Fix typos and improve grammar in Query Guide

### PR DESCRIPTION
**Pull request by AndreyBozhko**
https://issues.apache.org/jira/browse/SOLR-XXXXX

# Description

Made a pass over the Query Guide documentation, to improve the grammar and fix typos.

# Solution

Please provide a short description of the approach taken to implement your solution.

# Tests

Please describe the tests you've developed or run to confirm this patch implements the feature or solves the problem.

# Checklist

Please review the following and check all that apply:

- [x] I have reviewed the guidelines for [How to Contribute](https://wiki.apache.org/solr/HowToContribute) and my code conforms to the standards described there to the best of my ability.
- [ ] I have created a Jira issue and added the issue ID to my pull request title.
- [x] I have given Solr maintainers [access](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to contribute to my PR branch. (optional but recommended)
- [x] I have developed this patch against the `main` branch.
- [x] I have run `./gradlew check`.
- [ ] I have added tests for my changes.
- [ ] I have added documentation for the [Reference Guide](https://github.com/apache/solr/tree/main/solr/solr-ref-guide)

### CODE REVIEW

**Review by epugh — COMMENTED** [review]
This looks great!  Are you doing this by reading through the doc files or do you have some tooling helping you?

**Comment by epugh** [comment]
I triggered the precommit test.  Please tag me when you are ready for review and commit!

**Comment by AndreyBozhko** [comment]
Thanks for the review @epugh.

Majority of these are highlighted by IntelliJ in the Problems tool window, after it runs the inspection for the project or for specific files. Although, some of the typos definitely catch my eye when I read the documentation online.

**Comment by AndreyBozhko** [comment]
Hi @epugh - bumping this PR for your review.

**Comment by mkhludnev** [comment]
@epugh if you are busy I'd like to push it.
@AndreyBozhko it's awesome. Thanks.

**Comment by AndreyBozhko** [comment]
Hi @epugh - could you please take a look once more?

File: solr/solr-ref-guide/modules/query-guide/pages/common-query-parameters.adoc [hunk 1]
```diff
@@ -277,7 +277,7 @@ For example:
 
 [source,text]
 ----
-q=supervillians&debugQuery=on&explainOther=id:juggernaut
+q=supervillains&debugQuery=on&explainOther=id:juggernaut
```
Comment (epugh) [inline solr/solr-ref-guide/modules/query-guide/pages/common-query-parameters.adoc:280]: Just a quick, this query works now?    Wanted to make sure the typo doesn't actually need to be there!

**Review by epugh — APPROVED** [review]
LGTM.   I flagged one typo in a query, but I think jstut o double check that the fix was needed!   This was a huge amount of work, thank you!

File: solr/solr-ref-guide/modules/query-guide/pages/common-query-parameters.adoc (hunk 1 above)
Comment (epugh) [reply solr/solr-ref-guide/modules/query-guide/pages/common-query-parameters.adoc:280]: Never mind, I check, and now I get it!

**Comment by epugh** [comment]
Thank you @mkhludnev and @AndreyBozhko  for pinging me on this.  Really great to see our docs getting some :love:

**Pull request merged**